content/id/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: "Memperluas Kubernetes"
```

Comment:
According to mas ivan lanin's glosarium, "Memperluas" is the right translation to "extend", great job!